GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -18,7 +18,11 @@ public MacPlatform()
                 underConstruction: new UnderConstructionFlags(
                     supportsGVFSService: false,
                     supportsGVFSUpgrade: false,
-                    supportsGVFSConfig: false))
+                    supportsGVFSConfig: false),
+                diskLayoutVersion: new DiskLayoutVersion(
+                    currentMajorVersion: 18,
+                    currentMinorVersion: 0,
+                    minimumSupportedMajorVersion: 7))

```

Comment:
`minimumSupportedMajorVersion` should be bumped to 18 as well so that `TryCheckDiskLayoutVersion` returns the correct error message.